### PR TITLE
Make internal users vars generic and refactor i18n labels

### PR DESCRIPTION
**Pull request by sergibondarenko**
This closes https://floragunn.atlassian.net/browse/ITT-2124

### CODE REVIEW

File: public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js [hunk 1]
```diff
@@ -278,7 +274,7 @@ class InternalUsers extends Component {
     ];
 
     const selection = {
-      selectable: user => user.id,
+      selectable: resource => resource.id,
```
Comment (mgustafsson1) [inline public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js]: Not sure, but probably check for reserved here?

Comment (sergibondarenko) [reply public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js]: It was already solved in this PR https://github.com/floragunncom/search-guard-kibana-plugin/pull/198

Comment (sergibondarenko) [reply public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js]: Resolved the conflict in the commit below.

**Comment by sergibondarenko** [comment]
I resolved the conflicts and merged this PRs changes manually.

**Pull request closed without merging**